jetty-openid/src/main/config/modules/openid.mod

```diff
@@ -0,0 +1,34 @@
+DO NOT EDIT - See: https://www.eclipse.org/jetty/documentation/current/startup-modules.html
+
+[description]
+Adds OpenId Connect authentication.
+
+[depend]
+security
+
+[lib]
+lib/jetty-openid-${jetty.version}.jar
+lib/jetty-util-ajax-${jetty.version}.jar
+
+[files]
+basehome:modules/openid/openid-baseloginservice.xml|etc/openid-baseloginservice.xml
+
+[xml]
+etc/openid-baseloginservice.xml
+etc/jetty-openid.xml
+
+[ini-template]
+## The OpenID Identity Provider
+# jetty.openid.openIdProvider=https://accounts.google.com/
+
+## The Client Identifier
+# jetty.openid.clientId=1051168419525-5nl60mkugb77p9j194mrh287p1e0ahfi.apps.googleusercontent.com
```

Comment:
Is this some fake ID or a real one? You don't want to put here a real ID!

Reply:
These were real for my testing app, but I will change to fake ones.
The client secret can be reset easily so didn't think it was a big deal if it was real while testing.

Reply:
... and now that a real one has been published, you will need to revoke that from your account, otherwise anybody can start using your account for auth!  Low risk I know, but good practise to always cancel any credentials that have been published accidentally.

Reply:
Client secrets have been reset.
Except for on Yahoo because the site is so bugged I cannot reset the secret, delete the app, or contact support without errors.